packages/elements/src/components/ino-select/ino-select.tsx

```diff
@@ -227,7 +227,7 @@ export class Select implements ComponentInterface {
             />
           </div>
           <div class="mdc-select__menu mdc-menu mdc-menu-surface mdc-menu-surface--fullwidth">
-            <ul class="mdc-list" ref={(el) => (this.mdcOptionsListEl = el)}>
+            <ul class="mdc-deprecated-list" ref={(el) => (this.mdcOptionsListEl = el)}>
```

Comment:
Do we need a follow up to move away from deprecated list?

Reply:
https://github.com/inovex/elements/issues/541